specification/cloudshell/resource-manager/Microsoft.Portal/stable/2018-10-01/cloudShell.json

```diff
@@ -0,0 +1,952 @@
+{
```

Comment:
Are both cloudShell.json and CloudShell.json intended in the PR? I reviewed CloudShell.json, not cloudShell.json

Reply:
Just CloudShell.json. Removed cloudShell.json